client/signup/jetpack-connect/jetpack-new-site/index.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import page from 'page';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import config from 'config';
+import { recordTracksEvent } from 'state/analytics/actions';
+import SiteURLInput from '../site-url-input';
+import ReaderBack from '../../../blocks/reader-full-post/back';
+
+import WordPressLogo from 'components/wordpress-logo';
+import JetpackLogo from 'components/jetpack-logo';
+
+class JetpackNewSite extends Component {
+	componentDidMount() {
+		this.props.recordTracksEvent( 'calypso_jetpack_new_site_view' );
+	}
+
+	getNewWpcomSiteUrl() {
+		return config( 'signup_url' ) + '?ref=calypso-selector';
+	}
+
+	onURLEnter( url ) {
+		this.props.recordTracksEvent( 'calypso_jetpack_new_site_connect_click' );
+		page( '/jetpack/connect?url=' + url );
+	}
+
+	handleOnClickTos() {
+		this.props.recordTracksEvent( 'calypso_jpc_tos_link_click' );
+	}
+
+	handleBack( event ) {
+		event.preventDefault();
+
+		// @todo
+		// - Record action that user went back
+		// - Go back in history
+	}
+
+	render() {
+		return (
+			<div>
+				<ReaderBack onBackClick={ this.handleBack } />
+				<div className="jetpack-new-site__main jetpack-new-site">
+					<div className="jetpack-new-site__header">
+						<h2 className="jetpack-new-site__header-title">{ this.props.translate( 'Add a New Site' ) }</h2>
```

Comment:
Hi! I've found a possible matching string that has already been translated **66** times:
`translate( 'Add New Site' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-sites-new)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).